var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -188,14 +196,15 @@ def install(self, spec, prefix):
             options.extend(['-DTPL_ENABLE_HDF5:BOOL=OFF'])
 
         # Fortran lib
-        libgfortran = os.path.dirname(os.popen(
-            '%s --print-file-name libgfortran.a' %
-            join_path(mpi_bin, 'mpif90')).read())
-        options.extend([
-            '-DTrilinos_EXTRA_LINK_FLAGS:STRING=-L%s/ -lgfortran' % (
-                libgfortran),
-            '-DTrilinos_ENABLE_Fortran=ON'
-        ])
+        if self.compiler.name == 'gcc':
```

Comment:
this will most likely break it for clang+gfortran, for now please add another if with `clang`. You can also write it shorter by `spec.satisfies('%clang')` and the same for `gcc`.